CHANGELOG.md

```diff
@@ -5,29 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.5.0] - 2022-06-01
-
-### Changed
-
-- Modified sortby and collections parameters in nextlink
-
-## [0.5.0] - 2022-05-20
-
-### Changed
-
-- Used map instead of foreach/push in api.js file
-
-## [0.5.0] - 2022-05-18
+## [Unreleased]
 
 ### Added
 
 - Added STAC_API_VERSION as an environment variable to the serverless.yml file
-- Added STAC_API_VERSION to display on the API landing page within the api.js file
-under src/lib/api.js in the collectionsToCatalogLinks function
+- Added STAC_API_VERSION to display on the API landing page within the api.js file under src/lib/api.js in the collectionsToCatalogLinks function
+- Add support for pre- and post-hooks
 
 ### Changed
 
 - Changed the rel type to 'server' for the URL to the STAC API webpage inside the Links object
+- Modified sortby and collections parameters in nextlink
+- Used map instead of foreach/push in api.js file
```

Comment:
Could you add a note in Changed about not compressing the  response and the 6MB limit?